docs/workers.md

```diff
@@ -418,6 +418,23 @@ all these to be folded into the `generic_worker` app and to use config to define
 which processes handle the various proccessing such as push notifications.
 
 
+## Migration from old config
```

Comment:
I'd have split this up and had two separate "migration" sections, but this is fine.